### PR TITLE
Create customer data policy

### PR DESCRIPTION
**Pull request by ebrodymoore**

### CODE REVIEW

**Comment by ebrodymoore** [comment]
@dadlerj reminder on this

File: handbook/bizops/customer_data_policy.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+|----------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Pings from self\-hosted instances                                    | Delete data for any instance that's been offline for over 3 years\. All active instance data are kept                                                                                                        | For \(but not limited to\) business cases: 1\) History of previous instances for customer and prospect conversations 2\) understand how growth patterns within accounts evolve over many years |
```
Comment (dadlerj) [inline handbook/bizops/customer_data_policy.md:3]:
```suggestion
| Pings from self\-hosted instances                                    | Delete data for any instance that's been offline for over 3 years\. All active instance data are kept                                                                                                        | For \(but not limited to\) business cases: 1\) sales and customer success conversations 2\) providing technical support 3\) new release and security update notifications 4\) analyzing aggregate product usage and growth trends  |
```

File: handbook/bizops/customer_data_policy.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+| Sourcegraph\.com data \(events and users\)                           | Retain data for 1 year; retain aggregated daily usage numbers for all data; keep customer account unless requested to be deleted                                                                             | Retain 1 year of granular to make Sourcegraph\.com product/business decisions                                                                                                                  |
```
Comment (dadlerj) [inline handbook/bizops/customer_data_policy.md:4]:
```suggestion
| Sourcegraph\.com profile data (username, email, profile settings, saved searches, etc.) | Retain indefinitely unless deletion request is received | To provide uninterrupted product access and to retain settings and alerts for users who choose to return and use Sourcegraph.com |
| Sourcegraph\.com user action/event logs | Retain data for 1 year | For product usage analysis |
| Sourcegraph\.com aggregate daily user counts and anonymous traffic data | Retain data indefinitely | For tracking long-term usage and growth |
```

File: handbook/bizops/customer_data_policy.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+| HubSpot \(CRM/marketing automation\)                                 | Retain all customer information                                                                                                                                                                              | For context with customer conversations and do not contact lists                                                                                                                               |
+| Looker                                                               | n/a                                                                                                                                                                                                          | Looker only queries the database and doesn't store data                                                                                                                                        |
```
Comment (dadlerj) [inline handbook/bizops/customer_data_policy.md:6]: If we don't store data here, it doesn't need to be on the list
```suggestion
```

File: handbook/bizops/customer_data_policy.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+| Slack                                                                | Customer support channels are maintained; all other messages are deleted after 3 months                                                                                                                      | Customer support channels are kept for context and to better diagnose future issues                                                                                                            |
```
Comment (dadlerj) [inline handbook/bizops/customer_data_policy.md:7]:
```suggestion
| Slack                                                                | Customer support channels are maintained indefinitely; all other messages are deleted after 3 months                                                                                                                      | Customer support channels are kept for context and to better diagnose future support issues                                                                                                            |
```

**Comment by ebrodymoore** [comment]
Thanks for comments!
> I don't think you need backslashes for punctuation or all those spaces for the markdown table to work, by the way...

I use a md table formatter on the Internet 😅

**Pull request merged**